packages/playground/remote/service-worker.ts

```diff
@@ -6,13 +6,32 @@ import { getURLScope, removeURLScope } from '@php-wasm/scopes';
 import { applyRewriteRules } from '@php-wasm/universal';
 import {
 	awaitReply,
-	convertFetchEventToPHPRequest,
+	convertFetchEventToPHPRequest as convertFetchEvent,
 	initializeServiceWorker,
 	cloneRequest,
 	broadcastMessageExpectReply,
 } from '@php-wasm/web-service-worker';
 import { wordPressRewriteRules } from '@wp-playground/wordpress';
 
+async function convertFetchEventToPHPRequest(event: any) {
+	const fullUrl = new URL(event.request.url);
+	const scope = getURLScope(fullUrl);
+	const res: any = await convertFetchEvent(event);
+	const contentType: string = res.headers.get('content-type');
+	if (contentType?.match(/text\/html/)) {
+		const canonical: string = await res.text();
+		let relative = canonical.replace(
+			/(http|https):[\/\\]+(localhost|127.0.0.1|playground\.wordpress\.net|diy-pwa\.com)[:0-9]*\/scope:\d.\d*/g,
```

Comment:
We shouldn't have anything domain-specific in the converter. Why do these domains need to be threaded differently from other domains?